client/src/Components/TaskEstimatedAdminBox/TaskEstimatedAdminBox.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { forwardRef } from 'react';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+
+import { useBackend } from 'hooks/useBackend';
+import { TaskEstimatedAdminBoxStyles } from './TaskEstimatedAdminBoxStyles';
+import { useUserContext } from 'Contexts/UserContext';
+
+import { TextField, Button } from '@material-ui/core';
+
+const Schema = yup.object().shape({
+  result: yup.number().required(),
+});
+
+const TaskEstimatedAdminBox = ({ onClose }) => {
+  const sendVotesShow = useBackend('SUBMIT_ESTIMATION');
+  const sendVotesClear = useBackend('CLEAR_VOTES');
+  const { isAdmin } = useUserContext().user;
+  const { register, handleSubmit, errors } = useForm({
+    validationSchema: Schema,
+  });
+  const classes = TaskEstimatedAdminBoxStyles({ errors });
+
+  const onHandleClick = ({ result }) => {
+    sendVotesShow({ result });
+    sendVotesClear();
```

Comment:
Czy po zasubmitowaniu estymacji musimy jeszcze czyścić głosy osobnym socketem? Jeśli tak to wygląda na poprawkę do zrobienia na backendzie.